billing.go

```diff
@@ -23,7 +23,7 @@ type Billing struct {
 	IPAddressCountry string   `xml:"ip_address_country,omitempty"`
 
 	// Credit Card Info
-	FirstSix int    `xml:"first_six,omitempty"`
+	FirstSix string `xml:"first_six,omitempty"` // Card number can start with zero
```

Comment:
Minor nit, but just for consistency please update this comment to match the one for `LastFour`. That way it's even more obvious to see that these are both strings for the same reason.

`// String not int so that leading zeros are present`

Reply:
updated